parser/tsx.js

```diff
@@ -8,11 +8,9 @@
 
 'use strict';
 
-const _ = require('lodash');
 const babylon = require('@babel/parser');
-const baseOptions = require('./tsOptions');
-
-const options = _.merge(baseOptions, { plugins: ['jsx'] });
+const options = require('./tsOptions');
+options.plugins.push('jsx');
```

Comment:
This will mutate `baseOptions.plugins`, since it's the same array reference, which I don't think is what we want.

I think you want to instead clone `baseOptions` and add the `jsx` plugin... Something like this:
```js
const baseOptions = require('./tsOptions');
const options = {
  ...baseOptions,
  plugins: [...baseOptions.plugins, 'jsx'],
);
```

However we might have to use basic JS here (ie. no object spreads) for compatibility with older Node.js versions, so maybe something like this:
```js
const baseOptions = require('./tsOptions');
const options = Object.assign({}, baseOptions);
options.plugins = ['jsx'].concat(baseOptions.plugins);
```

Please test it though :)

Reply:
As I mentioned above, the existing code was using `_.merge` which also mutated `baseOptions`.